specs/0117-email-opt-in-unsubscribe.md

```diff
@@ -0,0 +1,130 @@
+- Button: `No, thank you`
+- Button (primary): `Enable email updates`
+- Consent: `By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+
+## Additional card for tour
+
+As part of new user onboarding, we point new users to user preferences and call out the ability to enable email communication preferences.
+
+(Section: content)
+- Header: `Stay up to date`
+- Body: `You can receive updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform by email. If you'd like to stay up to date, you can enable these emails in your settings.`
+
+*Display if user logged in:*
+- Button: `Enable email updates`
+- Consent: `By enabling email updates, I’m okay with Mozilla handling my personal information as explained in this <a href="https://www.mozilla.org/privacy/websites/">Privacy Notice</a>.`
+
+## Unsubscribe page
+
+Legal regulations (e.g. CAN-SPAM act) have specific requirements to include an unsubscribe link in commercial email messages to manage email preferences without requiring a login.
+
+As a user I can:
+- Access my unique/personal unsubscribe page through a link without logging in. The link is typically included with non-transactional emails from Pontoon as an “Unsubscribe” link in the footer.
+  - Link technical specs:
+  1) Pontoon generates a random unique identifier and assigns it to each user. The algorithm doesn't use any existing user data to generate the ID, which needs to be sufficiently complex to avoid a potential attacker guessing.
+  2) The unsubscribe page can be accessed without login. If a user ID is not provided, or it's incorrect, it will display an error message.
+  3) Unsubscribe links in emails will include the user's unique identifier in the URL to correctly identify the user.
+- Access all options from a single page; I do not need to navigate from the page linked in the email to manage subscriptions.
+
+(Section: content)
+ - Title: `Mozilla Localization - Pontoon`
+ - Header: `Update Your Email Preferences`
+ - Body: `Thank you for contributing to localization at Mozilla. Please use the form below if you would like to stop receiving email updates related to localization at Mozilla, projects on Pontoon, and updates to the Pontoon platform.`
+ - Radio button: `Please unsubscribe from all Mozilla localization emails.`
+ - Radio button: `Please keep sending me emails from Mozilla localization.`
+ - Button: `Submit`
+
+ - On submit alert (Unsubscribe): `You've successfully unsubscribed. You can update your preferences anytimes from your <a href="https://pontoon.mozilla.org/settings/">settings</a>.`
+ - On submit alert (Subscribe): `Thanks! You will continue to receive emails from Mozilla localization. You can update your preferences anytimes from your <a href="https://pontoon.mozilla.org/settings/">settings</a>.`
```

Comment:
If we go the checkbox route, we could have a message like "If you don't want to unsubscribe, or you have clicked the link by mistake, you can close this page and we will make no changes to your settings.`.